code/controllers/subsystem/SSthrowing.dm

```diff
@@ -93,6 +118,48 @@ SUBSYSTEM_DEF(throwing)
 	/// Will thrownthing datum actually block movement? this might be FALSE with some circumstances even if var/should_block_movement is TRUE. This variable change automatically during the throw
 	var/block_movement = TRUE
 
+/datum/thrownthing/New(thrownthing, atom/target, init_dir, maxrange, speed, atom/thrower, diagonals_first, force, callback, dodgeable, block_movement, target_zone)
+	. = ..()
```

Comment:
Is there actually a parent/behavior that this is calling?

Reply:
Yes. `/atom/New(loc, ...)` in `atoms.dm` line 117.

Reply:
this is a cargo cult moment